setup.py

```diff
@@ -17,7 +17,7 @@
       ],
       url="https://github.com/transferwise/pipelinewise-singer-python",
       install_requires=[
-          'pytz==2020.1',
+          'pytz >=2019, <=2021',
```

Comment:
can we use 'pytz <=2021' to be it in sync with https://github.com/snowflakedb/snowflake-connector-python/blob/8e2cd7235df7bd1dca71419f2bf9ab67e98c5484/setup.py#L199 ?

Reply:
Makes perfect sense to me, I won't be able to update until tonight but can hit it then

Reply:
updated ✅